README.md

```diff
@@ -1,24 +1,170 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# 📗 _Table of Contents_
 
-Things you may want to cover:
+- [📖 About the Project](#about-project)
+  - [🛠️ Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Database](#database)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* Ruby version
+# Budget App <a name="about-project"></a>
 
-* System dependencies
+Budget App is an expense tracking app that allows user to create new transactions associated with a category and see the money spent on each category.
 
-* Configuration
+- [Video Presentation](https://www.loom.com/share/ecdac42a8968406da79079bcc89a968e?sid=d7847d9a-241a-4e3d-8466-27c6d39aa4b7)
 
-* Database creation
+## :hammer_and_wrench: Built With <a name="built-with"></a>
 
-* Database initialization
+### Tech Stack
 
-* How to run the test suite
+<details><summary>Client</summary>
+<ul>
+  <li><a href="https://html.com/">HTML</a></li>
+  <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+</ul>
+</details>
+<details><summary>Backend</summary>
+<ul>
+  <li><a href='https://rubyonrails.org/'>Rails</a></li>
+</ul>
+</details>
+<details><summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
 
-* Services (job queues, cache servers, search engines, etc.)
+### Key Features
 
-* Deployment instructions
+- Record a transaction
+- Associate a transaction with a category
+- Create new categories
+- View amount spent on each category
 
-* ...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## :rocket: Live Demo
+
+- [Live Demo]( )
+
```

Comment:
- [ ] Kindly deploy your app and add its link to the readme file.

- [ ] Kindly record a video presentation and add its link to the readme file.